_doc-build-windows/action.yml

```diff
@@ -379,20 +379,17 @@ runs:
     # ------------------------------------------------------------------------
 
     - uses: ansys/actions/_logging@main
-      if: ${{ inputs.add-pdf-html-docs-as-assets == 'true' }}
```

Comment:
Could you remind me why did we put this condition check in the first place?

Reply:
Previously we didn't have `EXPECTED_BUILD_DIR` or `EXISTS_EXPECTED_BUILD_DIR` environment variables.
They were introduced when the feature to add assets was added through `add-pdf-html-docs-as-assets`.

Since some projects can provide a different build directory, the environment variables scope of work was associated to the asset feature which requires a standard location (or would have to ask the user for the location). Since we try to share common design, this was a lever for adoption :)

Reply:
OK, now I remember it. Thanks